leshan-core/src/main/java/org/eclipse/leshan/core/node/LwM2mResourceInstance.java

```diff
@@ -224,7 +225,7 @@ public String toString() {
         // We don't print OPAQUE value as this could be credentials one.
         // Not ideal but didn't find better way for now.
         return String.format("LwM2mResourceInstance [id=%s, value=%s, type=%s]", id,
-                type == Type.OPAQUE ? ((byte[]) value).length + "Bytes" : value, type);
+                LwM2mValueUtil.toPrettyPrint(type, value), type);
```

Comment:
is `toPrettyPrint` instead of `toPrettyString` on purpose ?

Reply:
:facepalm: